### DISCUSSION TITLE
Help updating embyserver to latest beta

### DISCUSSION THREAD

**Discussion by ricostuart** [tteck/Proxmox#3402] [Help Needed] [answered]
I've tried to download and ammend the update script to look for the lastest update to get the latest beta release but keeps failing. Can anyone help?

**Comment by Node815** [comment, marked as answer]
This what I do, I just manually grab the release page:
https://emby.media/linux-server.html
Then, I choose "Debian" 
Then I scroll down to the beta, and right click on the link (in the case of this writing) 

I log into the Emby container using the PVE console link and run `wget https://github.com/MediaBrowser/Emby.Releases/releases/download/4.9.0.27/emby-server-deb_4.9.0.27_amd64.deb ` (which is the latest as of now). 

Then, I run `systemctl stop emby-server.service ` if you are already in the console to stop the server.  Then I run:

`dpkg -i emby-server-deb_4.9.0.27_amd64.deb` (using the above filename)

This will run the install script and auto start the service.  It's been my go to process for every time there's a beta update which is quite frequent for them.
